src/System.Diagnostics.EventLog/tests/EventLogEntryCollectionTests.cs

```diff
@@ -10,6 +11,18 @@ public class EventLogEntryCollectionTests
     {
         private const string message = "EntryCollectionMessage";
 
+        private void WaitForEventLog(EventLog eventLog, int entriesExpected)
```

Comment:
Not limited to Windows 7?

Reply:
one failure was on windows10amd also
https://mc.dot.net/#/user/KKhurin/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/05e96931232ee754fe1438edecaec45e1aecbdfa/workItem/System.Diagnostics.EventLog.Tests/analysis/xunit/System.Diagnostics.Tests.EventLogEntryCollectionTests~2FCheckingEntryEqualityAndIndex

Reply:
OK, should you move this to Helpers, with the other retry wrappers?